packages/typescript-to-proptypes/src/parser.ts

```diff
@@ -186,12 +186,6 @@ export function parseFromProgram(
       return t.createObjectType({ jsDoc: undefined });
     }
 
-    const defaultGenericType = type.getDefault();
```

Comment:
We would no longer use the default value but the actual value passed to the generic or the constraint if no value is passed.